src/main/java/net/mcreator/ui/modgui/ModElementChangedListener.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.modgui;
+
+import javafx.embed.swing.JFXPanel;
+import net.mcreator.ui.component.JItemListField;
+import net.mcreator.ui.minecraft.JEntriesList;
+import net.mcreator.ui.minecraft.MCItemHolder;
+
+import javax.swing.*;
+import javax.swing.event.ChangeEvent;
+import javax.swing.event.ChangeListener;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import javax.swing.text.JTextComponent;
+import java.awt.*;
+import java.awt.event.*;
+
+public interface ModElementChangedListener
+		extends MouseListener, KeyListener, ActionListener, ChangeListener, DocumentListener {
+
+	default void registerUI(JComponent container, boolean addIdleListeners) {

```

Comment:
Please some documentation on what addIdleListeners means

Reply:
Like this?

Reply:
Are the docs OKay?